### DISCUSSION TITLE
proxy not working... still able to see the request trough network tab on safari

### DISCUSSION THREAD

**Discussion by ndcg91** [GEDYSIntraWare/cordova-plugin-webview-proxy#19] [Help]
Hi
I am trying to integrate this plugin to overcome a CORS error while developing my app. 
Somehow the plugins send the request but I can see the request trough network tab on safari so... I ended up having the same issue as before...CORS restrictions.

I DO see the request that change from _https_proxy_ to the actual url but at the end the request is sent trough the webview instead of been native. 

Can you please help?

here you have an example of my implementation 

     var url = window.cordova ? window.WebviewProxy.convertProxyUrl(config.services.host + config.services.logout2ndStep) : config.services.host + config.services.logout2ndStep;
                        $http({
                                method: 'POST',
                                url: url,
                                withCredentials: true
                            })

**Comment by NiklasMerz** [comment]
Did you enable the custom scheme with the preferences?

```
<preference name="scheme" value="app" />
<preference name="hostname" value="testapp"/>
```
